pkg/providers/aws/provider_postgres.go

```diff
@@ -35,8 +37,9 @@ import (
 )
 
 const (
-	postgresProviderName       = "aws-rds"
-	DefaultAwsIdentifierLength = 40
+	CRO_AWS_RDS_SERVICE_MAINTENANCE = "cro_aws_rds_service_maintenance"
```

Comment:
It looks like the best practice in golang, is to use `camelCase`, even for `consts`.